Server/Server App/Server App/Globals.cs

```diff
@@ -10,7 +10,7 @@ namespace Server_App
     internal static class Globals
     {
         private static SqlConnection? dbConnection = null;
-
+        public static string client_username;
```

Comment:
The global client username already exists on the main branch and is set to ThreadStatic. So please remove this one